### DISCUSSION TITLE
Deploying your jobs

### DISCUSSION THREAD

**Discussion by motdde** [timgit/pg-boss#343] [Q&A]
In a monolith, do you separate your jobs to a different server?

What the impact on performance when you running your jobs on the same server with your application?

**Comment by timgit** [comment]
In my application, I run a separate worker container just for background jobs.  The impact of running it in the same process is very situational to what your jobs are doing.  For example, it may have a negative impact if you use shared resources between your jobs and another application, such as a web API.  A popular use case is trying to share a database connection pool between jobs and the API.

**Reply by motdde** [reply]
Thanks, Tim.
